### PR TITLE
fix compilation

### PR DESCRIPTION
**Pull request by pizzadude**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@ set (nx_VERSION_MAJOR 2)
 set (nx_VERSION_MINOR 1)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_SOURCE_DIR}/bin)
```
Comment (isage) [inline CMakeLists.txt]: Why?

Comment (pizzadude) [inline CMakeLists.txt]: Without doing that:

git clone https://github.com/isage/nxengine-evo
cd nxengine-evo
cmake .
make

Scanning dependencies of target extract
[  0%] Building CXX object CMakeFiles/extract.dir/extract/main.cpp.o
[  1%] Building CXX object CMakeFiles/extract.dir/extract/extractstages.cpp.o
[  2%] Building CXX object CMakeFiles/extract.dir/extract/extractpxt.cpp.o
[  2%] Building CXX object CMakeFiles/extract.dir/extract/extractfiles.cpp.o
[  3%] Building CXX object CMakeFiles/extract.dir/extract/crc.cpp.o
[  4%] Building CXX object CMakeFiles/extract.dir/common/misc.cpp.o
[  5%] Building CXX object CMakeFiles/extract.dir/common/stat.cpp.o
[  5%] Building CXX object CMakeFiles/extract.dir/stagedata.cpp.o
[  6%] Linking CXX executable extract
/usr/bin/ld: cannot open output file extract: Is a directory
collect2: error: ld returned 1 exit status
CMakeFiles/extract.dir/build.make:276: recipe for target 'extract' failed
make[2]: *** [extract] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/extract.dir/all' failed
make[1]: *** [CMakeFiles/extract.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2

Comment (isage) [inline CMakeLists.txt]: eh, nobody (usually) does that.
```
mkdir build
cd build
cmake ..
make
```

Comment (isage) [inline CMakeLists.txt]: Though, I'll accept it, it may prove usefull for packaging

**Pull request merged**